hybris/custom/klarnapayment/src/com/klarna/payment/converter/populator/KlarnaExpCheckoutAuthPayloadPopulator.java

```diff
@@ -444,7 +443,7 @@ private Long calculateOrderEntryTaxAmount(final AbstractOrderEntryModel entry, f
 	private Long calculateTotalAmount(final AbstractOrderModel source)
 	{
 		LogHelper.debugLog(LOG, "entering KlarnaExpCheckoutAuthPayloadPopulator.calculateTotalAmount ... ");
```

Comment:
No debug logging in production code

Reply:
Debug logs have been removed
Commit can be found in https://github.com/klarna/SAP-Commerce-klarna-payments/commit/c3d029e9830e600544e03277455cf385023f4a5c